zetta_utils/builder/__init__.py

```diff
@@ -11,3 +13,8 @@
 from . import built_in_registrations
 
 PARALLEL_BUILD_ALLOWED: bool = False
+
+NP_MANUAL: List[str] = []
+for k in dir(np):
```

Comment:
Numpy methods are a bit strange... it's a wild mix of `np._ArrayFunctionDispatcher`, `np.ufunc` and regular methods, so we can't use the same trick that worked for [ torch ](https://github.com/ZettaAI/zetta_utils/blob/main/zetta_utils/convnet/architecture/primitives.py#L28)

Looking at https://github.com/numpy/numpy/issues/24019 the recommended way seems to be `inspect.isroutine`. I would use that as well as skip all `k.startswith('_')`.

You might also want to register methods from submodules `np.linalg` and `np.fft`, similar to how we did it for torch